fenix/app/src/main/res/values/strings.xml

```diff
@@ -2213,10 +2213,16 @@
     <string name="review_quality_check_contextual_onboarding_caption" moz:RemovedIn="121" tools:ignore="UnusedResources">By selecting “Yes, try it” you agree to %1$s by Mozilla’s %2$s and %3$s.</string>
     <!-- Caption text to be displayed in review quality check contextual onboarding card above the opt-in button. First parameter is the Fakespot product name. Following parameters are for clickable texts defined in review_quality_check_contextual_onboarding_privacy_policy and review_quality_check_contextual_onboarding_terms_use. -->
     <string name="review_quality_check_contextual_onboarding_caption_2">By selecting “Yes, try it” you agree to %1$s’s %2$s and %3$s.</string>
```

Comment:
lint is failing, would need --> `moz:RemovedIn="121" tools:ignore="UnusedResources"`